lib/Macros/Sources/ObservationMacros/ObservableMacro.swift

```diff
@@ -417,12 +479,20 @@ extension ObservationTrackedMacro: PeerMacro {
       return []
     }
     
-    if property.hasMacroApplication(ObservableMacro.ignoredMacroName) ||
-       property.hasMacroApplication(ObservableMacro.trackedMacroName) {
+    if property.hasMacroApplication(ObservableMacro.ignoredMacroName) {
       return []
     }
+    let file = context.location(of: property)?.file
+    let filePath = file?.as(StringLiteralExprSyntax.self)?.segments.compactMap { (item) -> String? in
+      switch item {
+      case .stringSegment(let seg): seg.content.text
+      default: nil
+      }
+    }.first
```

Comment:
We shouldn't make any assumption on what the location nodes are. The proposal states:
> Source locations are described in an abstract form that can be interpolated into source code (they are expressions) in places that expect a string literal (for the file name) or integer literal (for line and column). As with makeUniqueName returning a TokenSyntax rather than a String, this abstraction allows the compiler to introduce a different kind of syntax node (that might not even be expressible in normal Swift) to represent these values.

The syntax tree for `property` also isn't necessarily the original file, so using SourceLocationConverter here isn't actually valid. You'll want to run `context.location` on the first statement in each accessor and then use the nodes given back as-is.